lib/airbrake/rails.rb

```diff
@@ -19,7 +19,7 @@ class Railtie < ::Rails::Railtie
             ActionDispatch::DebugExceptions,
             Airbrake::Rack::Middleware
           )
-        elsif defined?(::ActiveRecord)
+        elsif defined?(::ActiveRecord) && defined?(::ActiveRecord::ConnectionAdapters)
```

Comment:
Maybe we could get rid of `defined?(::ActiveRecord)` in favour of the new check? The new check covers the first one anyway.

Reply:
hmm.. ahh didn't realize that..   maybe we should just be fully explicit then and do the whole class name.  Thus removing any possibility of it being incorrect.   I'll make that adjustment.

Reply:
@kyrylo updated with a fixup.  And it looks like something is messed up with the circleci test rig as it's not installing certain gems